lib/iknow_params/parser.rb

```diff
@@ -144,11 +142,16 @@ def remove_blanks(arg)
     end
   end
 
-  # Add default parse methods for each basic serializer class
-  ObjectSpace.each_object(IknowParams::Serializer.singleton_class).each do |serializer_class|
-    name = serializer_class.name.demodulize.underscore
-    define_method(:"parse_#{name}_param") do |param, default: PARAM_REQUIRED|
-      parse_param(param, with: serializer_class, default: default)
+  # Allow serializers to register themselves
+  def self.register_serializer(name, serializer)
+    define_method(:"parse_#{name.underscore}_param") do |param, default: PARAM_REQUIRED|
```

Comment:
Would be nice to have a test to check the name mangling for more interesting names.